aiven_db_migrate/migrate/pgmigrate.py

```diff
@@ -731,6 +731,7 @@ def __init__(
         *,
         source_conn_info: Union[str, Dict[str, Any]],
         target_conn_info: Union[str, Dict[str, Any]],
+        pgbin: Optional[Path] = None,
```

Comment:
but now you have to change this type as well, otherwise you're still passing a str instead of a Path :-)

I recommend you manually run ```mypy``` on single files when you're changing them even though it's not enabled repository-wide.

Everything else should be ok.

Reply:
Thank you @alanfranz-at-work ! great tip about `mypy`!